packages/core/src/handlers/http.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application} from '../application';
+import {ServerRequest, ServerResponse} from 'http';
+import {Handler} from '../internal-types';
+
+export function httpHandler(app: Application): Handler {
+  return async (req: ServerRequest, res: ServerResponse): Promise<void> => {
+    await app.handleHttp(req, res);
+  };
```

Comment:
FWIW, this is completely unnecessary. `app.handleHttp` is intentionally defined via an arrow function (see [here](https://github.com/strongloop/loopback-next/blob/master/packages/core/src/application.ts#L60-L61)) to allow users to pass it around as a plain function (not a class member).

The code here can be simplified to the following:

```ts
export function httpHandler(app: Application): Handler {
  return app.handleHttp;
}
```